docs/source/daml/intro/2_Scenario.rst

```diff
@@ -0,0 +1,146 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Scenarios
+=========
+
+In this section you will test the ``Token`` model from :doc:`1_Token` using DAML's inbuilt ``scenario`` language. You'll learn about the basic features of scenarios:
+
+- Getting parties
+- Submitting transactions
+- Creating contracts
+- Testing for failure
+- Archiving contracts
+- Viewing ledger and final ledger state
+
+For this section, start a new module called ``Intro_2_Scenario`` and copy across the ``Token`` contract from :doc:`1_Token`. Alternatively, use the :download:`supplied source file <daml/Intro_2_Scenario.daml>`.
+
+.. scenario_basics:
+
+Scenario Basics
```

Comment:
You could have three sub-headings here, for a little more structure: Declaring a scenario (straight after the code snippet), Setting up a party (after one paragraph), and Submitting a transaction (after With a variable alice of type Party)

Reply:
I don't like too much structure as content becomes sparse. This is intended for sequential reading so I want a good amount of content under each heading.